build/types/core

```diff
@@ -17,6 +17,8 @@
 +../../lib/deprecate/enforcer.js
 +../../lib/deprecate/version.js
 
++../../lib/drm/playready.js
```

Comment:
If we have drm namespace now, maybe we should move DrmEngine, DrmUtils & FairPlayUtils there? As a separate commit of course.

Reply:
Yes, DrmEngine and DrmUtils make sense, FairPlayUtils too, but the problem is that we export that class... so I would need the change to be feat! and we don't want that for now.